docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '3'
+
+services:
+  app:
+    build: 
+      context: ./docker/go
+    container_name: planit-api
+    volumes: 
+      - .:/var/www
+    ports:
+      - "8080:8080"
```

Comment:
let's use variable ${APP_PORT}
it helps us to serve multiple app instances in the same serer (e.g for staging and production environment)

Reply:
However It's a basic setup so it would be fine :)

Reply:
- [x] use app_port env variable